docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/jira.md

```diff
@@ -0,0 +1,659 @@
+      commentID: "${{ outputs['previous-comment-step'].commentID }}"
+```
+
+## Status Tracking
+
+### Wait for Status
+
+Waits for a Jira issue to reach a specific status before proceeding.
+
+#### Configuration
+
+| Name                           | Type     | Required | Description                                                    |
+| ------------------------------ | -------- | -------- | -------------------------------------------------------------- |
+| `waitForStatus.issueKey`       | `string` | Y        | The Jira Issue Key (e.g., EXT-123).                            |
+| `waitForStatus.expectedStatus` | `string` | Y        | The expected status to wait for (e.g., 'IN PROGRESS', 'DONE'). |
+
+#### Output
+
+This step does not produce any output values.
+
+#### Example
+
+This example waits for a change request issue to be approved before proceeding with promotion.
+
+```yaml
+steps:
+- uses: jira
+  config:
+    credentials:
+      secretName: jira-credentials
+    waitForStatus:
+      issueKey: "${{ freightMetadata(ctx.targetFreight.name, 'change-request-key') }}"
+      expectedStatus: "Approved"
+# promotion steps continue after approval...
+- uses: helm-template
+  config:
+    path: ./charts
+    vars:
+      imageTag: "${{ imageFrom(vars.imageRepo).Tag }}"
+      environment: "${{ ctx.stage }}"
+```
+
+:::info Content Formatting
+The Jira configuration supports setting issue and comment content using `description` or `body` fields. These are plain text fields that do not support special formatting such as Markdown. For rich formatting capabilities, use the ADF (Atlassian Document Format) alternatives `adfDescription` or `adfBody`. For more information about ADF structure and formatting, see the [Atlassian Document Format documentation](https://developer.atlassian.com/cloud/jira/platform/apis/document/structure/).
+:::
+
+## Multi-Stage Workflow
+
+The Jira promotion step automatically stores created issue keys in the Freight metadata, allowing subsequent stages to reference the same issue. This enables tracking a single Jira issue across multiple promotion stages.
+
+### Accessing Issue Keys from Freight Metadata
+
+Use the `freightMetadata()` template function to retrieve issue keys stored by previous stages:
+
+```yaml
+# Access the default issue key
+issueKey: ${{ freightMetadata(ctx.targetFreight.name, 'jira-issue-key') }}
+
+# Access a custom issue key (when issueAlias was used during creation)
+issueKey: ${{ freightMetadata(ctx.targetFreight.name, 'my-custom-alias') }}
+```
+
+### Example
+
+This comprehensive example demonstrates using the Jira promotion step across multiple stages in a promotion pipeline, tracking a single issue from its creation during testing through its final promotion to production:
+
+```yaml
+---
+apiVersion: kargo.akuity.io/v1alpha1
+kind: Stage
+metadata:
+  name: test
+  namespace: kargo-proj
+spec:
+  requestedFreight:
+  - origin:
+      kind: Warehouse
+      name: nginx
```

Comment:
I suggested the project name `GUESTBOOK` in a few places, so I wonder if we should make the Warehouse name and the image repo it subscribes to match.

It _feels_ slightly better imho because the idea of moving the latest revision of homegrown software through a pipeline probably resonates with most readers more readily than the idea of moving a piece of infra through a pipeline (even though that's a perfectly valid thing to do).

I intend to update the quickstart similarly for the same reason.

Just a thought. Up to you if you think this is worth it or not.


```suggestion
      name: nginx
```